src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -220,6 +221,18 @@ private function getContainerDefinitionData(Definition $definition, bool $omitTa
             'autoconfigure' => $definition->isAutoconfigured(),
         );
 
+        if (interface_exists('phpDocumentor\Reflection\DocBlockFactoryInterface')) {
```

Comment:
we should add a `protected getClassDescription` or so in the base `Descriptor` class and reuse it :)

Reply:
done